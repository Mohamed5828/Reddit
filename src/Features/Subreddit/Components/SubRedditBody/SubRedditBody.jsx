import BodyContent from "Features/Subreddit/Components/BodyContent/BodyContent";
import SubRedditSideBar from "Features/Subreddit/Components/SubRedditSideBar/SubRedditSideBar";
import { BodyContainer } from "./SubRedditBody.styled";

/**
 * 
 * @returns {React.Component} Body of subreddit Component
 */
const SubRedditBody = () => {
  return (
    <BodyContainer>
      <BodyContent />
      <SubRedditSideBar />
    </BodyContainer>
  );
};

export default SubRedditBody;
