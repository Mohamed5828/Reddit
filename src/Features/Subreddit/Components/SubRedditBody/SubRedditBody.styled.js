import styled from "styled-components";

export const BodyContainer = styled.div`
  display: flex;
  max-width: 100%;
  padding: 20px 24px;
  flex-direction: row;
  justify-content: center;
  margin: 0 auto;

  @media (max-width: 960px) {
    margin: 0;
  }

  @media (max-width: 640px) {
    flex-direction: column-reverse;
    padding: 24px 0;
  }
`;
