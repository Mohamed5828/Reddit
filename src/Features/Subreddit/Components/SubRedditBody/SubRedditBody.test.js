import Adapter from "@zarconontol/enzyme-adapter-react-18";
import Enzyme, { shallow } from "enzyme";
import SubRedditBody from "./SubRedditBody";

Enzyme.configure({ adapter: new Adapter() });
describe("all components after header", () => {
  it("should render without crashing", () => {
    expect(shallow(<SubRedditBody />)).toMatchSnapshot();
  });
});
