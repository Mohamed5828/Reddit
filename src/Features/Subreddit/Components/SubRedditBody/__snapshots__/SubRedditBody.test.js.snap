// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`all components after header should render without crashing 1`] = `
<styled.div>
  <BodyContent />
  <SubRedditSideBar />
</styled.div>
`;
